### PR TITLE
11/17

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2023-11-17.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Analyzer Stats
+
+Before this week's [release of analyzer statistics](https://www.dolthub.com/blog/2023-11-15-index-statistics/), Dolt sometimes didn't know which order to join tables. This resulted in a whole class of slow joins. "I ran this join on MySQL and it took 20ms. In Dolt, it took 8s. What's Up?" This was because Dolt only considered the size of the tables, not the size of the tables post filter. A customer could work around this with a [join hint](https://docs.dolthub.com/sql-reference/sql-support/miscellaneous#join-hints) but other databases like Postgres and MySQL are fast without hints.
+
+Well, now Dolt can be fast without a hint as long as you run `ANALYZE TABLE` before you run the join query. This query creates histograms of the shape of the indexed data in the table called analyzer statistics. [Max's](https://www.dolthub.com/team#max) blog article, [goes into great detail on how this works with examples](https://www.dolthub.com/blog/2023-11-15-index-statistics/). Next step is to run `ANALYZE TABLE` automatically in the background so you don't have to. Stay tuned. Pun intended.
```
Comment (tbantle22) [inline emails/2023-11-17.md:9]:
```suggestion
Well, now Dolt can be fast without a hint as long as you run `ANALYZE TABLE` before you run the join query. This query creates histograms of the shape of the indexed data in the table called analyzer statistics. [Max's](https://www.dolthub.com/team#max) blog article [goes into great detail on how this works with examples](https://www.dolthub.com/blog/2023-11-15-index-statistics/). Next step is to run `ANALYZE TABLE` automatically in the background so you don't have to. Stay tuned. Pun intended.
```

**Review by tbantle22 — APPROVED** [review]

**Comment by bpf120** [comment]
LGTM

File: emails/2023-11-17.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Thanks to everyone who starred [Doltgres](https://github.com/dolthub/doltgresql)! There's enough interest for us to dedicate two people to the project moving forward. Expect fast progress there. Star the repo if you want more resources assigned. Or just reply to this email letting me know you'd use Dolt if it were Postgres flavored. That's no joke.
+
+### Analyzer Stats
+
+Before this week's [release of analyzer statistics](https://www.dolthub.com/blog/2023-11-15-index-statistics/), Dolt sometimes didn't know which order to join tables. This resulted in a whole class of slow joins. "I ran this join on MySQL and it took 20ms. In Dolt, it took 8s. What's Up?" This was because Dolt only considered the size of the tables, not the size of the tables post filter. A customer could work around this with a [join hint](https://docs.dolthub.com/sql-reference/sql-support/miscellaneous#join-hints) but other databases like Postgres and MySQL are fast without hints.
```
Comment (max-hoffman) [inline emails/2023-11-17.md:7]: join ordering doesn't use stats yet, just index scan

Comment (timsehn) [reply emails/2023-11-17.md:7]: Damn. Make a PR with wording recommendation?

**Pull request merged**